### PR TITLE
chore: ffmpeg package in the voice doc

### PR DESCRIPTION
**Pull request by Helmasaur**
**Please describe the changes this PR makes and why it should be merged:**

ffmpeg package changed from `ffmpeg-binaries` to `ffmpeg-static`.

**Status**
- [ ] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
- [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: docs/topics/voice.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Voice in discord.js can be used for many things, such as music bots, recording o
 In discord.js, you can use voice by connecting to a `VoiceChannel` to obtain a `VoiceConnection`, where you can start streaming and receiving audio.
 
 To get started, make sure you have:
-* FFmpeg - `npm install ffmpeg-binaries`
+* FFmpeg - `npm install ffmpeg-static`
 * an opus encoder, choose one from below:
   * `npm install node-opus` (better performance)
```
Comment (amishshah) [inline docs/topics/voice.md]:
```suggestion
  * `npm install @discordjs/opus` (best performance)
  * `npm install node-opus`
```

Can you also make this change while you are editing this file?

Comment (iCrawl) [reply docs/topics/voice.md]: No this should probably be a separate PR. I'll cover this.

**Comment by Helmasaur** [comment]
Should those modifications be done in the stable branch too?

**Comment by iCrawl** [comment]
No

**Pull request merged**